### ISSUE TITLE
Training modernbert vs deberta

### ISSUE THREAD

**Issue by nico2rdj** [urchade/GLiNER#293]
Has anyone experimented with finetuning ModernBERT? I'm experiencing fast overfitting when training ModernBERT, but with DeBERTa, the validation loss keeps improving over many epochs
